### PR TITLE
zendesk fix

### PR DESCRIPTION
**Pull request by guyfreund**
## Status
Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/20876

## Description
Fixing the issue without breaking BC

## Does it break backward compatibility?
   - No

## Must have
- [x] Code Review

## Technical Writer Review
- [x] [YML](https://github.com/demisto/content/blob/c1b87393ce483d4641e478a41c373c06e6cf831a/Integrations/integration-Zendesk.yml) - description in lines 786-787 needs to be reviewed

### CODE REVIEW

**Comment by kirbles19** [comment]
@guyfreund Done. Why is there no changelog.md ?

**Comment by guyfreund** [comment]
@kirbles - There’s no chanelog since this bug has been reopened and logs for this bug are already in the changelog file. When I’ll resolve the conflicts it’ll appear.

**Comment by guyfreund** [comment]
Since we don't have an instance for development, the fix was tested on zendesk production env.
As can see down below, the fix solves the issue and doesn't break BC.
image

File: Integrations/integration-Zendesk_CHANGELOG.md [hunk 1]
```diff
@@ -1,2 +1,5 @@
 ## [Unreleased]
- -
\ No newline at end of file
+-
+
+## [19.12.1] - 2019-12-25
+Fixed an issue where the ***zendesk-add-user*** command didn't return an error when a user is already exists.
```
Comment (ShahafBenYakir) [inline Integrations/integration-Zendesk_CHANGELOG.md]: Why is this on previous release?

**Comment by guyfreund** [comment]
@kirbles19 - Please review changelog

**Comment by kirbles19** [comment]
@guyfreund Done.

**Comment by kirbles19** [comment]
@guyfreund @ShahafBenYakir We have no documentation for this integration at all. Please generate documentation (MD) for the entire integration.

**Comment by guyfreund** [comment]
@kirbles19 Please open a documentation issue.

**Comment by kirbles19** [comment]
I thought all docs are supposed to be part of the integration package.

**Comment by ShahafBenYakir** [comment]
@kirbles19 It is, but in this case, as you mentioned, no docs yet exist and thus someone should write them from scratch i.e. a new task which is not related to this bug fix made by @guyfreund.

**Comment by kirbles19** [comment]
@ShahafBenYakir Actually, technically, there was an addition made which requires updates to documentation, therefore docs are definitely related to the PR. Docs should've been generated without me having to request it.

**Comment by guyfreund** [comment]
@kirbles19 If the docs already existed, then this PR is supposed also to update the docs. But since they do not exist, the docs is another issue that has to be done from scratch, therefore has to prioritized.

**Comment by kirbles19** [comment]
@guyfreund So create the docs, and i'll create an issue for it. It's a high priority, if the integration was important enough to update, it's important enough to have documentation for.

What's the ETA?

**Pull request merged**